doc/Overview/Validation/ValidationOverview.md

```diff
@@ -1,16 +1,92 @@
 ---
 uid: Overview.Validation
 ---
+
 # Validation
 
-Two options:
-- Class can inherit from ObservableValidator and use attributes on properties to define validation rules (eg [Required])
-- Can use [FluentValidation](https://www.nuget.org/packages/FluentValidation/) to execute validation rules. Check out the section on [Dependency Injection](https://docs.fluentvalidation.net/en/latest/di.html#) on how to configure validators
+Uno.Extensions.Validation provides an `IValidator` service to validate that properties on an object follow a set of characteristics or behaviors defined by [attributes](https://learn.microsoft.com/dotnet/api/system.componentmodel.dataannotations) applied to them. This is useful for ensuring that data entered by users is valid before it is saved to a database or sent to a web service.
+
+## Service registration
+
+An extension method for `IHostBuilder` is provided to register the `IValidator` service with the DI container.
+
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e){
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseValidation();
+        });
+...
+```
+
+## Validation service
+
+It can then be injected into any class that needs to validate data. The `ValidateAsync` method is used to execute validation rules on an object. The method returns a `ValueTask` that completes when validation is complete. The `ValueTask` result is a `IEnumerable<ValidationResult>` which contains elements which indicate whether the object is valid or not.
+
+## Validation-capable classes
+
+Instances passed into the `ValidateAsync` method should implemement `INotifyDataErrorInfo`. It follows that there are two primary options for making a class validation-capable:
+
+- Inherit from `ObservableValidator` and use attributes on properties to define validation rules such as `[Required]`
+- Use [FluentValidation](https://www.nuget.org/packages/FluentValidation/) to execute validation rules. Check out the [section](https://docs.fluentvalidation.net/en/latest/di.html#) on how to configure validators
+
```

Comment:
Rework the next section (perhaps using tabs) to show using validation with the three options